stdlib/streams/src/test/resources/test-src/throttler/time-batch.bal

```diff
@@ -0,0 +1,104 @@
+import ballerina/streams;
+import ballerina/time;
+
+public type TimeBatch object {
+    public string EXPIRY_TIME_STAMP;
```

Comment:
why the variable name is defined as capital characters?

Reply:
it has to be a const.. will fix

Reply:
it seems we cannot define const in objects.. so i changed it to a field variable